packages/block/block.go

```diff
@@ -136,6 +136,9 @@ func (b *Block) Play(dbTransaction *model.DbTransaction) error {
 			logger.WithFields(log.Fields{"type": consts.DBError, "error": err, "tx_hash": t.TxHash}).Error("using savepoint")
 			return err
 		}
+
+		model.IncrementTxAttemptCount(nil, t.TxHash)
```

Comment:
why not within DbTransaction?

Reply:
separated db tx in this case allow save changes in spit of execution result